docs/credstores.md

```diff
@@ -209,6 +209,9 @@ export GCM_CREDENTIAL_CACHE_OPTIONS="--timeout 300"
 git config --global credential.cacheOptions "--timeout 300"
 ```
 
+As for _Windows_, if you enable `git.exe credential-cache` on a Windows version that supports Unix sockets, it will work, otherwise it will error out and tell you why.
```

Comment:
I don't think the "otherwise it will error out and tell you why" part is true yet. Have you tried it?

Reply:
Sorry, I misunderstand that "otherwise it will error out and tell you why" part means showing "fatal: credential-cache unavailable; no unix socket support".
I have not been able to reproduce the cases where `credential-cache` fails if `NO_UNIX_SOCKETS` overridden.
So I remove this part. Is it OK?

Reply:
> So I think I just remove this part for now. Is it OK?

That would match my preference, too.

Reply:
Thank you for comment, I fixed again.
Please re-review.

Reply:
Thank you!

I would prefer it, though, if this PR did not introduce a change in one commit only to remove the same change in the next commit. Could I ask you to [squash (i.e. collapse) the first three commits into a single one](https://git-scm.com/docs/git-rebase#_interactive_mode)?